### PR TITLE
Update multisig tests and benches

### PR DESCRIPTION
**Pull request by vmarkushin**
Signed-off-by: Vladislav Markushin <negigic@gmail.com>

### CODE REVIEW

**Review by KalitaAlexey — APPROVED** [review]

File: multisig/src/weights.rs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+pub trait WeightInfo {
+    fn as_multi_create(s: u32, z: u32, ) -> Weight;
+    fn as_multi_create_store(s: u32, z: u32, ) -> Weight;
+    fn as_multi_approve(s: u32, z: u32, ) -> Weight;
+    fn as_multi_approve_store(s: u32, z: u32, ) -> Weight;
+    fn as_multi_complete(s: u32, z: u32, ) -> Weight;
+}
+
+/// Weights for pallet_multisig using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
```
Comment (modbrin) [inline multisig/src/weights.rs:54]: We probably need to rebenchmark those, or at least scale comparing to other weights.

File: multisig/src/benchmarking.rs [hunk 2]
```diff
@@ -272,16 +273,16 @@ mod tests {
     #[test]
     fn test_benchmarks() {
         new_test_ext().execute_with(|| {
-            assert_ok!(test_benchmark_as_multi_threshold_1::<Test>());
+            // assert_ok!(test_benchmark_as_multi_threshold_1::<Test>());
             assert_ok!(test_benchmark_as_multi_create::<Test>());
             assert_ok!(test_benchmark_as_multi_create_store::<Test>());
             assert_ok!(test_benchmark_as_multi_approve::<Test>());
             assert_ok!(test_benchmark_as_multi_complete::<Test>());
-            assert_ok!(test_benchmark_approve_as_multi_create::<Test>());
```
Comment (modbrin) [inline multisig/src/benchmarking.rs:280]: What's the problem with these tests?

**Review by modbrin — APPROVED** [review]

File: multisig/src/benchmarking.rs (hunk 2 above)
Comment (vmarkushin) [reply multisig/src/benchmarking.rs:280]: We don't use the `approve_as_*` functions, so I decided to not include them in benches

File: multisig/src/weights.rs (hunk 1 above)
Comment (vmarkushin) [reply multisig/src/weights.rs:54]: Yes

**Review by KalitaAlexey — APPROVED** [review]

**Pull request merged**